morphia/src/test/java/org/mongodb/morphia/TestDocumentValidation.java

```diff
@@ -144,7 +144,10 @@ private void checkValidation(final Document validator, final MappedClass mappedC
             .append("validationLevel", level.getValue())
             .append("validationAction", action.getValue());
 
-        assertEquals(expected, getValidation());
+        Document validation = getValidation();
```

Comment:
"autoIndexId" or something to that effect.  Showed up sometime after 3.3.11 which I'd been using to test with until rc0 was released.